### ISSUE TITLE
Make book cover focusable and clickable

### ISSUE THREAD

**Issue by manuellagrand** [readium/readium-desktop#424]
"read book" button is missing in drop-down book menu. we should add button to allow blind person to 
access at book more easier.
Currently they must open a modal window by "about this book" button in drop-down menu then open a book by "Read" button, it take too much time.

![image.png](https://images.zenhubusercontent.com/5d19d4e1599844363331bf70/a986a45c-ce8f-4dca-8988-bf6e4c1291e0)

**Comment by danielweck** [comment]
Actually, wouldn't it be better to make the cover image keyboard-focusable? (`tabindex`)
This would offer keyboard users direct access to the "read book" command, just like a mouse user clicking on the cover image.

**Comment by manuellagrand** [comment]
yes i tested both way to open a book (image keyboard-focusable and read button in drop down menu). Image keyboard-focusable is more easier direct access for blind person.
